al-jazari/roster.json

```diff
@@ -33,6 +33,10 @@
     {
         "name": "Vlada Solomiana",
         "github": "solo-vlada"
-     }
+     },
+     {
+      "name": "Ayoub Laissaoui",
+      "github": "NovaDA"
+   }
```

Comment:
Can you please indent everything inside the curly braces and make sure that the curly braces are aligned?